migrations/20180411200942_add_header_to_export_job.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddHeaderToExportJob extends AbstractMigration
+{
+
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $this->table('export_job')
+            ->addColumn('header_row', 'text',
+				['null' => true, 'limit' => 16777215, 'default' => null]
```

Comment:
I have no idea how long this can be. It's a representation of all attributes that the user is exporting for their posts with priority, labels, type, etc

Reply:
right now it's set the length for mysql's "Medium text" ...